ansible/environments/docker-machine/hosts.j2.ini

```diff
@@ -10,9 +10,12 @@ ansible ansible_connection=local
 controller0                 ansible_host={{ docker_machine_ip }}
 controller1                 ansible_host={{ docker_machine_ip }}
 
-[kafka]
+[kafkas]
 {{ docker_machine_ip }}     ansible_host={{ docker_machine_ip }}
```

Comment:
👍 Due to the complaints of resource consumption for local development I think it's good just have a single kafka container in a docker-machine deployment.